templates/topic/reports/a11y-1.json

```diff
@@ -0,0 +1,689 @@
+{
+  "@context": "https://wet-boew.github.io/vocab/context/2023/assessment-report-en.json-ld",
+  "@type": [
+    "earl:Assertion",
+    "acr:AssessmentReport"
+  ],
+  "dct:date": "2024-10-03",
+  "earl:subject": {
+    "@id": "_:subject",
+    "dct:references": [
+      "https://wet-boew.github.io/GCWeb/templates/topic/topic-en.html",
+      "https://wet-boew.github.io/GCWeb/templates/topic/topic-stackedheader-en.html"
+    ],
+    "@type": [
+      "earl:TestSubject",
+      "schema:WebPage"
+    ],
+    "dct:description": "Evaluation of the GC topic template in order to determine if it is aligned with our design and is compliant to our accessibility guideline when used as is without any special customization.",
+    "oa:hasState": [
+      {
+        "@type": "acr:MediaQueryList",
+        "acr:stateValue": "(min-width: 992px) and (max-width: 1199px)",
+        "_:iterator": 0
+      }
+    ],
+    "earl:pointer": {
+      "@type": "oa:CssSelector",
+      "@value": "main"
+    }
+  },
+  "dct:description": "Evaluation of the GC topic template in order to determine if it is aligned with our design and is compliant to our accessibility guideline when used as is without any special customization. The pages were assessed to ensure that they meet the Web Content Accessibility Guideline by manually testing each success criteria.",
+  "earl:result": [
+    {
+      "earl:test": "WCAG21:non-text-content",
+      "earl:subject": {
+        "@id": "_non-text-content_itm1",
+        "dct:isPartOf": "_:subject",
+        "@type": ["earl:TestSubject", "schema:WebPageElement"],
+        "dct:description": "Non-text content includes text alternative that serves the equivalent purpose."
```

Comment:
This is the description for the inner tested subject, not a description about your evaluation.

I think you should move this description  at line 54 where it contains an empty description.

The description that we should see here is a textual refinement of what image in this page that you have evaluated. For example, "Right aligned image in the introduction bloc"